test/runtests.jl

```diff
@@ -23,7 +23,7 @@ include("test_utils.jl")
     # Note: The output of the doctests currently requires a newer version of Julia
     # https://github.com/JuliaLang/julia/pull/34387
     # The doctests fail on x86, so only run them on 64-bit hardware
-    if VERSION >= v"1.5.0-DEV.163" && Sys.WORD_SIZE == 64
+    if v"1.6" >= VERSION >= v"1.7" && Sys.WORD_SIZE == 64
```

Comment:
The previous statement is always false
```suggestion
    if v"1.6" >= VERSION && Sys.WORD_SIZE == 64
```

Reply:
I meant to do `v"1.6" <= VERSION < v"1.7"` 🤦‍♀️